mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -61,6 +61,18 @@ var (
 
 const BYTE_SIZE = 2000
 
+// helper function to get current kubernetes context
```

Comment:
Can this function be moved to utils, looks like useful function which can be leveraged by any other command/component.

Reply:
pkg/utils/helpers.go should be good.